projects/1.20/assets/refurbished-furniture/refurbished_furniture/lang/zh_cn-base.json

```
@@ -0,0 +1,211 @@
+    "gui.refurbished_furniture.link_too_many": "链接过多",
+    "gui.refurbished_furniture.link_already_connected": "已连接",
+    "gui.refurbished_furniture.link_invalid_node": "无效节点",
+    "gui.refurbished_furniture.link_unpowerable": "链路无法供电",
+    "gui.refurbished_furniture.link_outside_area": "链路超出供电区域",
+    "gui.refurbished_furniture.connect_to_power": "确保方块从%s接收电力，可以使用%s进行连接。",
+    "gui.refurbished_furniture.electricity_generator": "发电机",
+    "gui.refurbished_furniture.jei_campfire_info": "继承自营火烹饪",
+    "gui.refurbished_furniture.progress": "%s %s %s",
+    "gui.refurbished_furniture.hold_for_details": "按住%s查看详情",
+    "gui.refurbished_furniture.shift": "SHIFT",
+    "gui.refurbished_furniture.show_all_categories": "显示全部",
+    "gui.refurbished_furniture.mail_box_limit": "邮箱数量已达上限",
+    "gui.refurbished_furniture.booting": "启动中……",
+    "gui.refurbished_furniture.sliceable": "可切片",
+    "gui.refurbished_furniture.placeable": "可放置",
+    "gui.refurbished_furniture.workbench.search_neighbours.off": "忽略相邻容器",
+    "gui.refurbished_furniture.workbench.search_neighbours.on": "包含相邻容器",
+    "gui.refurbished_furniture.experience_level": "%s/%s 级",
+    "gui.refurbished_furniture.experience_points": "%s 点",
+    "gui.refurbished_furniture.withdraw_experience": "提取经验",
+    "gui.refurbished_furniture.requires_power": "需要从%s供电",
+    "subtitle.refurbished_furniture.package_open": "纸箱：撕裂",
+    "subtitle.refurbished_furniture.chair_slide": "木头：滑动",
+    "subtitle.refurbished_furniture.doorbell_chime": "门铃：鸣响",
+    "subtitle.refurbished_furniture.electricity_generator_engine": "发电机",
+    "subtitle.refurbished_furniture.storage_jar_insert_item": "物品：移动",
+    "subtitle.refurbished_furniture.recycle_bin_engine": "碾压",
+    "subtitle.refurbished_furniture.ceiling_fan_spin": "风扇：旋转",
+    "subtitle.refurbished_furniture.lightswitch_flick": "开关：拨动",
+    "subtitle.refurbished_furniture.trampoline_bounce": "弹跳",
+    "subtitle.refurbished_furniture.trampoline_super_bounce": "强力弹跳",
+    "subtitle.refurbished_furniture.television_channel.colour_test": "高音调",
+    "subtitle.refurbished_furniture.television_channel.white_noise": "白噪音",
+    "subtitle.refurbished_furniture.television_channel.dance_music": "舞曲节拍",
+    "subtitle.refurbished_furniture.television_channel.villager_news": "新闻频道",
+    "subtitle.refurbished_furniture.television_channel.chirp_song": "鸟鸣为歌",
+    "subtitle.refurbished_furniture.television_channel.ocean_sunset": "LoFi节拍",
+    "subtitle.refurbished_furniture.television_channel.blocky_game": "钢琴音乐",
+    "subtitle.refurbished_furniture.television_channel.retro_song": "复古街机歌曲",
+    "subtitle.refurbished_furniture.frying_pan.place_ingredient": "啪嗒",
+    "subtitle.refurbished_furniture.frying_pan.break": "金属：撞击",
+    "subtitle.refurbished_furniture.frying_pan.hit": "金属：敲击",
+    "subtitle.refurbished_furniture.frying_pan.place": "金属：放置",
+    "subtitle.refurbished_furniture.frying_pan.step": "金属：脚步",

```

```suggestion
    "subtitle.refurbished_furniture.frying_pan.place": "金属：被放置",
    "subtitle.refurbished_furniture.frying_pan.step": "金属：脚步声",
```
经检查为放置平底锅和在平底锅上行走时的音效，故根据原版字幕更改